src/high-level/matrix.lisp

```diff
@@ -487,6 +487,7 @@ If :SQUARE is T, then the result will be restricted to the upper rightmost squar
                                   (tref matrix i j))))
         target))))
 ;;; Synonym for upper-triangular
+(declaim (ftype (function (matrix)) triu))
```

Comment:
these aren't necessarily `matrix` functions, just the implementation provided is

Reply:
If these were to be replaced with `t`, would that impair any optimizations?

Reply:
I don't think so. There's a lot of indirection already.